.gitignore

```diff
@@ -14,4 +14,6 @@ hs_err_pid*
 
 # intelliJ
 *.idea
-*.iml
\ No newline at end of file
+*.iml
+/bin/
```

Comment:
This and the spaces/tab issues come from my using eclipse.
I'll fix them in next iteration.